src/Java.Interop.Tools.JavaCallableWrappers/Java.Interop.Tools.JavaCallableWrappers.csproj

```diff
@@ -5,9 +5,16 @@
   </PropertyGroup>
   <Import Condition="Exists ('$(XAConfigPath)')" Project="$(XAConfigPath)" />
 
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(DotNetTargetFramework)</TargetFrameworks>
+  <PropertyGroup Condition=" '$(XABuild)' == 'true' ">
+    <TargetFramework>netstandard2.0</TargetFramework>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(XABuild)' != 'true' ">
```

Comment:
I'm also not sure how this *works*; consider xamarin/xamarin-android:

```sh
% % git grep '<XABuild'
Configuration.props:    <XABuildToolsPackagePrefixMacOS></XABuildToolsPackagePrefixMacOS>
Configuration.props:    <XABuildToolsPackagePrefixWindows></XABuildToolsPackagePrefixWindows>
Configuration.props:    <XABuildToolsPackagePrefixLinux></XABuildToolsPackagePrefixLinux>
Configuration.props:    <XABuildToolsPackagePrefix Condition=" '$(HostOS)' == 'Darwin' ">$(XABuildToolsPackagePrefixMacOS)</XABuildToolsPackagePrefix>
Configuration.props:    <XABuildToolsPackagePrefix Condition=" '$(HostOS)' == 'Windows' ">$(XABuildToolsPackagePrefixWindows)</XABuildToolsPackagePrefix>
Configuration.props:    <XABuildToolsVersion>34</XABuildToolsVersion>
Configuration.props:    <XABuildToolsFolder Condition="'$(XABuildToolsFolder)' == ''">34.0.0</XABuildToolsFolder>
Configuration.props:    <XABuildingForNetCoreApp>True</XABuildingForNetCoreApp>
```

There's no *obvious* "XABuild" property within xamarin-android.

If I instead search for just "XABuild" (no `<`), I get *many* more matches, but at a glance none look like they *could* become MSBuild properties, and `git grep '\<XABuild\>'` finds *nothing*.

How would this work?

(Which is somewhat doubly odd, as `Condition=" '$(XABuild)' == 'true' "` exists *now*.  Is it dead code?)